### DISCUSSION TITLE
Training nequip on pdb datasets

### DISCUSSION THREAD

**Discussion by shreshthtuli** [mir-group/nequip#335] [Q&A]
Hi. I am wondering if it is possible to train and test nequip on pdb datasets with pairs of molecules, such as https://data.sbgrid.org/dataset/684/. Any help is greatly appreciated :)

**Comment by Linux-cpp-lisp** [comment]
Hi @shreshthtuli ,

Thanks for your interest in our code!

We generally recommend converting all data into the extXYZ format, which can be easily done by constructing ASE `Atoms` objects somehow (https://wiki.fysik.dtu.dk/ase/ase/atoms.html) and writing them to an XYZ file with `ase.io.write`.  I'm not familiar with the dataset you've linked, but in general just remember that in the default setup MLIPs like Allegro learn a mapping between a specific geometry and an energy, so if that sounds applicable then you should be fine to go ahead with the default setup.
